spec-main/api-auto-updater-spec.ts

```diff
@@ -0,0 +1,92 @@
+import { autoUpdater } from 'electron'
+import { expect } from 'chai'
+import { ifit, ifdescribe } from './spec-helpers'
+
+ifdescribe(!process.mas)('autoUpdater module', function () {
+  describe('checkForUpdates', function () {
+    ifit(process.platform === 'win32')('emits an error on Windows when called the feed URL is not set', function (done) {
+      autoUpdater.once('error', function (error) {
+        expect(error.message).to.equal('Update URL is not set')
+        done()
+      })
+      autoUpdater.setFeedURL({url:''})
+      autoUpdater.checkForUpdates()
+    })
+  })
+
+  describe('getFeedURL', () => {
+    it('returns an empty string by default', () => {
+      expect(autoUpdater.getFeedURL()).to.equal('')
+    })
+
+    ifit(process.platform === 'win32')('correctly fetches the previously set FeedURL', function (done) {
+      const updateURL = 'https://fake-update.electron.io'
+      autoUpdater.setFeedURL({url: updateURL})
+      expect(autoUpdater.getFeedURL()).to.equal(updateURL)
+      done()
+    })
+  })
+
+  describe('setFeedURL', function () {
+    ifdescribe(process.platform === 'win32' || process.platform === 'darwin')('on Mac or Windows', () => {
```

Comment:
```suggestion
    ifdescribe(process.platform !== 'linux')('on Mac or Windows', () => {
```

Reply:
i prefer it to explicitly match the name of the test :)